packages/stark-ui/src/modules/route-search/components/route-search.component.html

```diff
@@ -0,0 +1,57 @@
+<div *ngIf="direction === 'left'">
+	<form class="route-search">
+		<button mat-mini-fab mat-icon-button class="search-button"
+				(click)="show()"
+				[matTooltip]="'STARK.ROUTE_SEARCH.ABOUT'| translate">
+			<mat-icon svgIcon="magnify" starkSvgViewBox>
+			</mat-icon>
+		</button>
+		<div *ngIf="!hide">
+			<mat-form-field floatLabel="never" class="path-input-field">
+				<input class="path-input"
+					   matInput
+					   [matTooltip]="'STARK.ROUTE_SEARCH.PLACEHOLDER'| translate"
+					   [matAutocomplete]="auto"
+					   [placeholder]="'STARK.ROUTE_SEARCH.PLACEHOLDER'| translate"
+					   [(ngModel)]="inputValue"
+					   [formControl]="pathCtrl">
```

Comment:
We should use either `ngModel` or `formControl` but not both... otherwise we might have some weird side effects and potential errors...